lib/membrane_http_adaptive_stream/hls.ex

```diff
@@ -188,20 +221,38 @@ defmodule Membrane.HTTPAdaptiveStream.HLS do
     #EXT-X-VERSION:#{@version}
     #EXT-X-TARGETDURATION:#{target_duration}
     """ <>
-      serialize_ll_hls_tags(supports_ll_hls?, target_partial_duration) <>
+      serialize_ll_hls_tags(supports_ll_hls?, target_partial_duration, can_skip_segments_duration) <>
       """
       #EXT-X-MEDIA-SEQUENCE:#{track.current_seq_num}
       #EXT-X-DISCONTINUITY-SEQUENCE:#{track.current_discontinuity_seq_num}
       #EXT-X-MAP:URI="#{track.header_name}"
-      #{serialize_segments(track)}
+      #{serialize_segments(track.segments, supports_ll_hls?: supports_ll_hls?, delta?: delta?)}
       #{if track.finished?, do: "#EXT-X-ENDLIST", else: serialize_preload_hint_tag(supports_ll_hls?, track)}
       """
   end
 
-  defp serialize_segments(track) do
-    supports_ll_hls? = Track.supports_partial_segments?(track)
+  defp should_generate_delta_playlist?(track) do
+    Track.supports_partial_segments?(track) &&
+      Enum.count(track.segments) > @min_segments_in_delta_playlist
+  end
```

Comment:
Since this is only used in one place, let's assign it to a variable there instead of creating another function. It's going to read better.